internal/taskfile/task.go

```diff
@@ -20,5 +20,5 @@ type Task struct {
 	Silent       bool
 	Method       string
 	Prefix       string
-	IgnoreError  bool `yaml:"ignore_error"`
```

Comment:
Given the suggestion made on `func (t *Task) UnmarshalYAML`, we should re-add this reflect tag.